MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -5578,12 +5582,6 @@ public string GetVideoTransposeDirection(EncodingJobInfo state)
             if (doOclTonemap)
             {
                 var tonemapFilter = GetHwTonemapFilter(options, "opencl", "nv12");
-                // enable tradeoffs for performance
-                if (!string.IsNullOrEmpty(tonemapFilter))
-                {
-                    tonemapFilter += ":tradeoff=1";
```

Comment:
Is this because the dovi tone mapping not available or the filter can automatically set tradeoff flag?

Reply:
The filter can do it automatically, so it is redundant.